.github/workflows/fuzzing.yml

```diff
@@ -24,41 +24,45 @@ jobs:
     - name: Install `cargo-fuzz`
       run: cargo install cargo-fuzz
     - uses: Swatinem/rust-cache@v2
+    - name: Restore Cached Corpus
+      uses: actions/cache/restore@v3
+      with:
+        key: corpus-cache
+        path: |
+          fuzz/corpus
     - name: Run fuzz_date for XX seconds
+      continue-on-error: true
```

Comment:
why this change?

Reply:
and the others
continue-on-error: true

Reply:
because the fuzz test like fuzz_date will panic on a huge number as input so I want every step in this workflow to be passed then I can make sure the cache strategy is working.

but right now I need your guidance to tell me which one needs the 
```bash
continue-on-error: true
```
which one don't need.

thank you.

Reply:
we need it for expr and date. the rest should pass